src/NUnitTestAdapter/packages.config

```diff
@@ -1,4 +1,4 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <packages>
-  <package id="NUnit.Core.Engine" version="3.0.0-beta-3" targetFramework="net35" />
+  <package id="NUnit.Core.Engine" version="3.0.0-beta-2" targetFramework="net35" />
```

Comment:
Sorry, didn't read through all of the comments before code-reviewing. Please disregard.